pkg/goDB/storage/gpfile/gpfile_test.go

```diff
@@ -181,19 +181,153 @@ func testRoundtrip(t *testing.T, enc encoders.Type) {
 	}
 }
 
+func TestInvalidMetadata(t *testing.T) {
+
+	os.RemoveAll("/tmp/test_db")
+	if err := os.MkdirAll("/tmp/test_db/0", 0755); err != nil {
```

Comment:
It might be useful to use `stretchr`'s `testify` library and use `assert` instead.

Reply:
Sure... I thought you had reservations against that dependency (because we're not using it anywhere else :-P).